modules/store/spec/store.spec.ts

```diff
@@ -471,4 +476,174 @@ describe('ngRx Store', () => {
       mockStore.dispatch(action);
     });
   });
+
+  describe('Meta Reducers', () => {
+    let metaReducerContainer: any;
+
+    beforeEach(() => {
+      metaReducerContainer = (function() {
+        function metaReducer1(reducer: ActionReducer<any, any>) {
+          return function(state: any, action: Action) {
+            return reducer(state, action);
+          };
+        }
+
+        function metaReducer2(reducer: ActionReducer<any, any>) {
+          return function(state: any, action: Action) {
+            return reducer(state, action);
+          };
+        }
+
+        return {
+          metaReducer1: metaReducer1,
+          metaReducer2: metaReducer2,
+        };
+      })();
+    });
+
+    it('should create a meta reducer for root and call it through', () => {
+      const metaReducerSpy: any = spyOn(
+        metaReducerContainer,
+        'metaReducer1'
+      ).and.callThrough();
+      setup({ counter1: 0, counter2: 1 }, [metaReducerContainer.metaReducer1]);
```

Comment:
Since we don't do anything with the counter values, I think it will be clearer to use 

```ts
setup({ }, [metaReducerContainer.metaReducer1]);
```